adagrams/game.py

```diff
@@ -1,11 +1,128 @@
+import random
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+    list_letter_pool =[]
+    for letter, number in LETTER_POOL.items():
+        letter_quantity = letter * number
+        list_letter_pool.append(letter_quantity)
+
+
+    letter_string = ''.join(list_letter_pool)
+   
+    # Available letters with frequencies list
+    available_letters = list(letter_string)
```

Comment:
In this function, it looks like you're building a list of lists in `list_letter_pool`. Later on, you flatten `list_letter_pool` into a single list of strings in `available_letters`.

I think it might be worth thinking about how to refactor this logic-- while we iterate through `LETTER_POOL.items()`, is there a way we can create `available_letters` (without `list_letter_pool`)?